deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -385,6 +392,7 @@ def get(self, request, name, stage=None):
                 "display_stopping_hosts": DISPLAY_STOPPING_HOSTS,
                 "project_name_is_default": project_name_is_default,
                 "project_info": project_info,
+                "remaining_capacity": json.dumps(remaining_capacity),
```

Comment:
Is `json.dumps()` needed?

Reply:
Yes, otherwise the Cluster Component does not render properly

Reply:
`remaining_capacity` is just a number. Why would it not work?

Reply:
This issue occurs when there is no Capacity. Since the calculation for remainingCapacity  is done Python it returns None when there is no Cluster. Javascript does not recognize None which then prevents the component from rendering correctly. `json.dumps()` converts None to null.